api/src/main/java/org/openmrs/module/appointments/service/impl/AppointmentVisitLocation.java

```diff
@@ -53,4 +53,4 @@ public String getFacilityName(String locationUuid) {
         String visitLocationName = sendGetRequest(baseUrl + locationUuid);
         return visitLocationName;
     }
-}
+}
```

Comment:
Is this really required to make HTTP call. You can get the location from OMRS LocationService.getLocationByUuid() is enough